### PR TITLE
replaced [block:image] with markdown

### PR DESCRIPTION
**Pull request by dmitrydao**

### CODE REVIEW

File: docs/Additional Information/workshop.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Ropsten Explorer: <a href="https://ropsten.explorer.chain.link/" target="_blank"
 - Ropsten LINK faucet: <a href="https://ropsten.chain.link" target="_blank">https://ropsten.chain.link</a>
 
 ## Slide: Add the LINK token to Metamask
-[block:code]
+```
```
Comment (StephenFluin) [inline docs/Additional Information/workshop.md:35]: You can't just replace [block:code] with three backticks, you need to actually take the code out of the JSON to make sure it shows up correctly. It's not supposed to look like JSON, it's just supposed to be the code.

**Review by StephenFluin — CHANGES_REQUESTED** [review]
Need to extract all the code from the JSON.

File: docs/Additional Information/workshop.md (hunk 1 above)
Comment (dmitrydao) [reply docs/Additional Information/workshop.md:35]: noted

**Review by StephenFluin — CHANGES_REQUESTED** [review]
Can you make sure the `alt` tags are more descriptive? Imagine a user that has an accessibility need. In an ideal world, the alt text should supply the important information from the image, but at least it should describe what's in the image.

File: docs/Keepers/register-upkeep.md [hunk 2]
```diff
@@ -24,12 +24,12 @@ The website will walk you through several steps.
 # 5 Steps to register and fund Upkeep on the Chainlink Keeper Network
 
 1. **Connect your wallet** with the button in the top right corner and choose a chain. The Chainlink Keeper Network currently supports Ethereum Mainnet or Kovan.
-  ![Metamask](/images/contract-devs/keeper/keeper-metamask.png)
+  ![Connect With Metamask](/images/contract-devs/keeper/keeper-metamask.png)
 
-1. **Click the `Register new upkeep` button**
```
Comment (StephenFluin) [inline docs/Keepers/register-upkeep.md:29]: Put this back to 1. Let the browser do numbering.

File: docs/Keepers/register-upkeep.md [hunk 3]
```diff
@@ -39,27 +39,24 @@ The website will walk you through several steps.
     >
     > Your balance will be charged LINK based on a 20% premium over the gas cost to `performUpkeep`. There's currently a ~80k gas overhead from the registry. The premium and overhead are not fixed and will change over time.
 
-
-
     The gas limit of the example counter contract should be set to 200,000.
    
- 
-1. **Press `Register upkeep`** and confirm the transaction in MetaMask
+4. **Press `Register upkeep`** and confirm the transaction in MetaMask
```
Comment (StephenFluin) [inline docs/Keepers/register-upkeep.md:44]: Numbering

File: docs/Keepers/register-upkeep.md [hunk 4]
```diff
@@ -24,12 +24,12 @@ The website will walk you through several steps.
 # 5 Steps to register and fund Upkeep on the Chainlink Keeper Network
 
 1. **Connect your wallet** with the button in the top right corner and choose a chain. The Chainlink Keeper Network currently supports Ethereum Mainnet or Kovan.
-  ![Metamask](/images/contract-devs/keeper/keeper-metamask.png)
+  ![Connect With Metamask](/images/contract-devs/keeper/keeper-metamask.png)
 
-1. **Click the `Register new upkeep` button**
-  ![Register New Upkeep](/images/contract-devs/keeper/keeper-register.png)
+2. **Click the `Register new upkeep` button**
```
Comment (StephenFluin) [inline docs/Keepers/register-upkeep.md:29]: numbering

**Review by StephenFluin — CHANGES_REQUESTED** [review]
Fix numbering and then good to go

**Review by StephenFluin — APPROVED** [review]

**Pull request merged**